src/App.tsx

```diff
@@ -1,10 +1,27 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 
-export const App: React.FC = () => (
-  <div className="App">
-    {/* Keep this input for tests */}
-    <input type="text" />
+export const App: React.FC = () => {
+  const [pressedKey, changingKey] = useState('Nothing was pressed yet');
```

Comment:
initial state should be empty. check if it's empty in template and than show 'Nothing was pressed yet'